CRM/Extendedreport/Form/Report/ExtendedReport.php

```diff
@@ -6349,9 +6350,9 @@ protected function alterFinancialType($value, &$row, $selectedField, $criteriaFi
    * @param $value
    * @param $row
    *
-   * @return array
+   * @return array|string
    */
-  protected function alterContributionStatus($value, &$row): array {
+  protected function alterContributionStatus($value, &$row) {
```

Comment:
if there is a $value then we get a string back but if no value then we get an array